pegasus/test/fixtures/fake_dashboard.rb

```diff
@@ -2,6 +2,26 @@
 require 'active_record'
 require 'sequel'
 
+# Monkey-patch the SQLite connection adapter to ignore MySQL-specific schema creation statements.
+module ActiveRecord
+  module ConnectionAdapters
+    module SQLite3
+      class SchemaCreation < AbstractAdapter::SchemaCreation
+        private
+
+        def add_table_options!(create_sql, options)
+          if (options_sql = options[:options])
+            options_sql.gsub!(/ENGINE=\w+/, '')
+            options_sql.gsub!(/DEFAULT CHARSET=\w+/, '')
+            options_sql.gsub!(/COLLATE=\w+/, '')
+            create_sql << " #{options_sql}"
+          end
+        end
+      end
+    end
+  end
+end
+
```

Comment:
This patch is necessary since in Rails 5 platform-specific options are now being added to the (previously platform-agnostic) schema.rb file, e.g.:

```
options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci"
```

Without explicitly filtering out these options, SQLite errors out when it tries to load the schema.

I plan to followup on this issue upstream in the future, but until then this workaround does the job.